sdk/wallet-adapter/site/pages/advanced/wallet-standard.mdx

```diff
@@ -0,0 +1,147 @@
+# Wallet Standard
+
+Browser extension wallets built for Sui use the [Wallet Standard](https://github.com/wallet-standard/wallet-standard/) to define. This is a cross-chain standard that defines how wallets can automatically be discovered and interacted with from dapps.
```

Comment:
Browser extension wallets built for Sui are defined using the Wallet Standard. This is a cross-chain standard that defines how wallets can automatically be discovered and interacted with from dApps.